src/topp/InternalCalibration.cpp

```diff
@@ -313,6 +317,13 @@ class TOPPInternalCalibration :
         exp_out.set2DData(ic.getCalibrationPoints(), CalibrationData::getMetaValues());
         mz_file.store(file_cal_lock_out, exp_out);
       }
+      if (!file_cal_lock_fail_out.empty())
+      {
+        LOG_INFO << "\nWriting UNmatched lock masses to mzML file '" << file_cal_lock_fail_out << "'." << std::endl;
```

Comment:
Ok. This was actually INtentional fpr emphasis but i can lowercase it.
-- 
Sent via mobile phone.

Am 17. Mai 2016 21:58:34 MESZ, schrieb Hannes Roest <notifications@github.com>:
>> @@ -313,6 +317,13 @@ class TOPPInternalCalibration :
>>          exp_out.set2DData(ic.getCalibrationPoints(),
>CalibrationData::getMetaValues());
>>          mz_file.store(file_cal_lock_out, exp_out);
>>        }
>> +      if (!file_cal_lock_fail_out.empty())
>> +      {
>> +        LOG_INFO << "\nWriting UNmatched lock masses to mzML file '"
><< file_cal_lock_fail_out << "'." << std::endl;
>
>here
>
>---
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/OpenMS/OpenMS/pull/1988/files/d3e070696936b8a2cc795b48cfa5cf3e503b535b#r63592457